app/src/main/java/com/moviefiend/torian/moviefiend/network/MovieInfo.java

```diff
@@ -0,0 +1,81 @@
+package com.moviefiend.torian.moviefiend.network;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.gson.annotations.SerializedName;
+
+public class MovieInfo implements Parcelable {
+    @SerializedName("id")
+    Integer mId;
```

Comment:
👍  also changed the `Float` to `float`.
Any idea why I remembered that I can only use Integer and Float and such in the conversion?